test/fixed/concatenation-spacing.php

```diff
@@ -20,6 +20,6 @@
           . '2'
       . '3';
 
-$string = '1' .
-          '2' .
-          '3';
+$string = '1'
+          . '2'
+          . '3';
```

Comment:
Shouldn't these two be aligned with the equal sign?

```
$string = '1'
        . '2'
        . '3';
```

Reply:
Maybe it should but we don't have any sniff like that... And what if concatenation is in let say `sprintf`:

```php
throw new \Exception(sprintf(
    'This is very very very long error message'
    . ' with some exception: %s',
    $e->getMessage()
));
```

There is so many cases. ScopeIndent from PHP_CodeSniffer doesn't include so many cases and my is super strict, and controversial in some places ;-) We can try, but ... hah